packages/devtools_app/lib/src/charts/flutter/flame_chart.dart

```diff
@@ -739,3 +803,74 @@ mixin FlameChartColorMixin {
     _unknownColorOffset = 0;
   }
 }
+
+/// [ExtentDelegate] implementation for the case where size and position is
+/// known for each list item.
+class _ScrollingFlameChartRowExtentDelegate extends ExtentDelegate {
+  _ScrollingFlameChartRowExtentDelegate({
+    @required this.nodeRects,
+    @required this.zoom,
+    @required this.chartStartInset,
+    @required this.chartWidth,
+  }) {
+    recompute();
+  }
+
+  List<Rect> nodeRects = [];
+
+  double zoom;
+
+  double chartStartInset;
+
+  double chartWidth;
+
+  void recomputeWith({
+    @required List<Rect> nodeRects,
+    @required double zoom,
+    @required double chartStartInset,
+    @required double chartWidth,
+  }) {
+    this.nodeRects = nodeRects;
+    this.zoom = zoom;
+    this.chartStartInset = chartStartInset;
+    this.chartWidth = chartWidth;
+    recompute();
+  }
+
+  @override
+  double itemExtent(int index) {
+    if (index >= length) return 0;
+    return nodeRects[index].width;
+  }
+
+  @override
+  double layoutOffset(int index) {
+    if (index >= length) return nodeRects.last.right;
+    return nodeRects[index].left;
+  }
+
+  @override
+  int get length => nodeRects.length;
+
+  @override
+  int minChildIndexForScrollOffset(double scrollOffset) {
+    return _lowerBoundIndex(scrollOffset);
```

Comment:
for consistency you should match the rounding behavior of the other minChildIndexForScrollOffset implementation.

Reply:
done.